lib/components/AlbumScreen/preset_chip.dart

```diff
@@ -0,0 +1,153 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:finamp/services/queue_service.dart';
+import 'package:get_it/get_it.dart';
+
+const _radius = Radius.circular(4);
+const _borderRadius = BorderRadius.all(_radius);
+const _height = 36.0;
+final _defaultBackgroundColour = Colors.white.withOpacity(0.1);
+final _queueService = GetIt.instance<QueueService>();
```

Comment:
Please always move the `GetIt` calls into a widget. You should move this one into `_PresetChipState`